src/vma/netlink/netlink_wrapper.cpp

```diff
@@ -357,9 +357,10 @@ int netlink_wrapper::get_neigh(const char* ipaddr, netlink_neigh_info* new_neigh
 		nl_object_get(obj); //Acquire a reference on a cache object. cache won't use/free it until calling to nl_object_put(obj)
 		neigh = (rtnl_neigh*) obj;
 		nl_addr* addr = rtnl_neigh_get_dst(neigh);
-		if (addr) {
+		int index = rtnl_neigh_get_ifindex(neigh);
+		if ((addr) && (index > 0)) {
```

Comment:
If index<=0 would you print a debug message?  (if index==0 may print errno as well).